src/Umbraco.Core/Services/IContentVersionService.cs

```diff
@@ -13,10 +14,19 @@ public interface IContentVersionService
     ///     Gets paginated content versions for given content id paginated.
     /// </summary>
     /// <exception cref="ArgumentException">Thrown when <paramref name="culture" /> is invalid.</exception>
+    [Obsolete("Use the async version instead. Scheduled to be removed in v15")]
     IEnumerable<ContentVersionMeta>? GetPagedContentVersions(int contentId, long pageIndex, int pageSize, out long totalRecords, string? culture = null);
 
     /// <summary>
     ///     Updates preventCleanup value for given content version.
     /// </summary>
+    [Obsolete("Use the async version instead. Scheduled to be removed in v15")]
     void SetPreventCleanup(int versionId, bool preventCleanup, int userId = -1);
+
+    ContentVersionMeta? Get(int versionId);
+    Task<Attempt<PagedModel<ContentVersionMeta>?, ContentVersionOperationStatus>> GetContentVersionsAsync(Guid contentId, string? culture, int skip, int take);
+    Task<Attempt<IContent?, ContentVersionOperationStatus>> GetAsync(Guid versionId);
+
+    Task<Attempt<ContentVersionOperationStatus>> SetPreventCleanupAsync(Guid versionId, bool preventCleanup, Guid userKey);
+    Task<Attempt<ContentVersionOperationStatus>> RollBackAsync(Guid versionId, string? culture, Guid userKey);
```

Comment:
We need to mark the PR as breaking because of these and mention what's breaking in the PR